src/elements/content-sidebar/additional-tabs/AdditionalTab.js

```diff
@@ -77,6 +77,8 @@ class AdditionalTab extends React.PureComponent<Props, State> {
                     alt={title}
                 />
             );
+        } else if (id && id < 0 && icon) {
+            TabIcon = icon;
```

Comment:
does `id` need to be less than 0 or can this block be combined with the above `if`?

im thinking if in the future teams can utilize `icon` or `iconUrl` depending on what's available to them. e.g. if the "share slack" integration changes from url to svg, then the logic in this file doesn't need to be modified to handle that behavior